app/views/inspectors/service-inspector-utils-extension.js

```diff
@@ -121,7 +121,19 @@ YUI.add('service-inspector-utils-extension', function(Y) {
       @param {Y.Node} container The container of the prompt.
     */
     showDestroyPrompt: function(container) {
-      container.one('.destroy-service-prompt').removeClass('closed');
+      var prompt = container.one('.destroy-service-prompt');
+      // Use the inspectorHeader as our canary to get the model as it should
+      // always exist.
+      var model = this.views.inspectorHeader.model;
```

Comment:
Because this is an extension for `service-inspector` I believe there should be the model available using `this.get('model')`